services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -151,12 +152,20 @@ open class HankeServiceImpl(
 
         postProcessAndSaveLogging(loggingEntryHolder, savedHankeEntity, userId)
 
-        return createHankeDomainObjectFromEntity(entity).also {
-            hankeKayttajaService.saveNewTokensFromHanke(it)
+        return createHankeDomainObjectFromEntity(savedHankeEntity).also {
+            initAccessForCreatedHanke(it, userId)
             hankeLoggingService.logCreate(it, userId)
         }
     }
 
+    private fun initAccessForCreatedHanke(hanke: Hanke, userId: String) {
+        val hankeId = hanke.id!!
+        val permissionAll = permissionService.setPermission(hankeId, userId, Role.KAIKKI_OIKEUDET)
+        logger.info { "Saving token for Hanke perustaja." }
+        hankeKayttajaService.createToken(hankeId, hanke.perustaja.toUserContact(), permissionAll)
```

Comment:
Perustaja doesn't need a token, since they're getting KAIKKI_OIKEUDET permissions directly. A token is only needed for activating the user by creating the permission row later when we know which user they're supposed to be.

Reply:
Fixed the implementation so that perustaja does not get a kayttajatunniste.